Firestore/Protos/CMakeLists.txt

```diff
@@ -157,6 +157,12 @@ add_custom_command(
     ${PROTOBUF_PYTHON}
 )
 
+add_custom_target(
```

Comment:
Your other targets have an explicit COMMENT (which makes the output a little nicer). I'd suggest adding one here too.

Reply:
Oh good point; never mind.

... Hm. "Generatoring"?  See line 168.

Reply:
Generatoring: it's fun!